firefox_decrypt.py

```diff
@@ -768,6 +779,10 @@ def parse_sys_args():
 
     args = parser.parse_args()
 
+    if args.tabular:
+        args.format = 'csv'
+        args.delimiter = '\t'
```

Comment:
Yes it does escape them (that's why I implemented this at all :wink:). I tried to add a test case for this but I didn't know how to edit the firefox password files.